src/cljs/org/broadinstitute/firecloud_ui/common/input.cljs

```diff
@@ -24,8 +24,11 @@
                                :style (merge (or (:style props) {})
                                         (when (:invalid @state)
                                           {:borderColor (:exception-red style/colors)}))
-                               :onChange #(swap! state dissoc :invalid)
+                               :onChange #(do (swap! state dissoc :invalid)
+                                              (when-let [x (:onChange props)]
+                                                (x (.. % -target -value))))
                                :defaultValue (:defaultValue props)
+                               :value (:value props)
```

Comment:
It wouldn't make sense to set both, but it's okay here to just pass them along and let the underlying control handle that case.